adobe/react-spectrum
# fix(#1231) fix(#1070) A11y: add minimum/maximum labels, use output for displayValue

Pressing the displayValue should focus the corresponding input.

Closes #1231
Relates to #1070

## ✅ Pull Request Checklist:

- [x] Included link to corresponding React Spectrum GitHub Issues [#1231](https://github.com/adobe/react-spectrum/issues/1231)  and [#1070](https://github.com/adobe/react-spectrum/issues/1070).

## 📝 Test Instructions:

1. open http://localhost:9003/?path=/story/rangeslider--custom-width
2. Mouse down on "Label"
3. Arrow keys should change value of "Minimum" slider thumb
4. Mouse down on display value text for the "Maximum" slider, to the right of the en dash.
5. Arrow keys should change value of "Maximum" slider thumb
6. Mouse down on display value text for the "Minimum" slider, to the left of the en dash.
7. Arrow keys should change value of "Minimum" slider thumb
8. Inspect accessibility properties for input within minimum slider thumb.
9. Accessibility name should be "Label Minimum"
10. Inspect accessibility properties for input within maximum slider thumb.
11. Accessibility name should be "Label Maximum"
12. use mouse to drag Minimum slider on top of Maximum slider, so that they both have the same value
13. Starting to drag the Minimum slider again, should select the Minimum slider, that was on top last.
14. use mouse to drag Maximum slider on top of Minimum slider, so that they both have the same value
15. Starting to drag the Maximum slider again, should select the Maximum slider, that was on top last.


## 🧢 Your Project:
